www/content/docs/home/getting-started.mdx

```diff
@@ -56,3 +57,24 @@ const Artworks = props => {
   )
 }
 ```
+
+In order to use Artsy's fonts, you'll also want to add our webfont file to the
+CSS in the header of your app:
+
+```
+// Example using Styled Components:
+import { createGlobalStyle, css } from "styled-components"
+
+const GlobalStyles = createGlobalStyle<P>`
+  @import url("https://webfonts.artsy.net/all-webfonts.css");
+`
+
+// React Helmet example:
+<Helmet>
+  <link
+    href="https://webfonts.artsy.net/all-webfonts.css"
+    rel="stylesheet"
+    type="text/css"
+  />
+</Helmet>
```

Comment:
I like the two different example approaches! both very modern. Note however that we use [react-head](https://github.com/tizmagik/react-head) in Reaction so might want to also include that, and put it above the React Head example.